addons/cluster-autoscaler/README.md

```diff
@@ -1,12 +1,14 @@
 # Cluster Autoscaler Addon
 
+Note that you likely want to change `AWS_REGION` and `GROUP_NAME`, and probably `MIN_NODES` and `MAX_NODES`
```

Comment:
Maybe something like: 

- Configure the following environment variables as appropriate for your deployment.